cli/functionary/package.py

```diff
@@ -96,7 +97,64 @@ def buildstatus(ctx, id):
     """
     if id:
         results = get(f"builds/{id}")
-        format_results([results], title=f"Build: {id}")
+        format_results([results], title=f"Build: {id}", excluded_fields=["environment"])
     else:
         results = get("builds")
-        format_results(results, title="Build Status")
+        format_results(results, title="Build Status", excluded_fields=["environment"])
+
+
+@package_cmd.command()
+@click.pass_context
+@click.option("--id", help="view a specific package with the given id")
+def view(ctx, id):
+    """
+    View all current packages and their functions
+    """
+    packages = get("packages")
+    functions = get("functions")
+    if id:
+        name = ""
+        description = ""
+        found = False
+        for package in packages:
+            if package["id"] == id:
+                name = package["name"]
+                description = package["description"]
+                found = True
+        if found:
+            associated_functions = _find_functions(id, functions)
+        else:
+            raise click.ClickException(f"Package with id {id} not found")
+    else:
+        for i in range(0, len(packages)):
+            package = packages[i]
+            name = package["name"]
+            id = package["id"]
+            description = package["description"]
+            associated_functions = _find_functions(id, functions)
+    title = Text(f"{name}", style="bold green")
+    title.append(f"\n{description}", style="green dim")
+    format_results(associated_functions, title=title)
```

Comment:
I'm pretty sure that as written this is only going to output the last package result rather than all of them.